ubuntudesign/documentation_builder/build.py

```diff
@@ -82,15 +112,28 @@ def build_files(self):
         parse all files into a new folder of HTML files
         """
 
-        if path.relpath(self.source_media_path, self.output_media_path) != '.':
-            mergetree(self.source_media_path, self.output_media_path)
-            print(
-                "Copied {} to {}".format(
-                    self.source_media_path, self.output_media_path
+        if path.isdir(self.source_media_path):
+            if path.relpath(
+                self.source_media_path, self.output_media_path
+            ) != '.':
```

Comment:
I think this would look cleaner if you created a variable first explaining the reasoning. Something vaguely along the lines of:
```
        if path.isdir(self.source_media_path):
            is_local_media_path = path.relpath(
                self.source_media_path, self.output_media_path
            ) == '.'
            if not is_local_media_path:
                ...
```